src/JSONPartition.ts

```diff
@@ -1,46 +1,44 @@
 import { Partition } from "./Partition";
 import { Note } from "./Note";
-
-interface JSONNote {
-  note: string;
-  duration: number;
-  octave: number;
-}
+import { CustomChord } from "./CustomChord";
+import { JSONNote, JSONMusic } from "./interface/JSONPlayable";
 
 class JSONPartition extends Partition {
   /**
-   * @param {JSONNote[]} json - The JSON that will be parsed into a partition.
+   * @param {JSONMusic[]} json - The JSON that will be parsed into a partition.
    * @param {OscillatorType} oscillator - The oscillator type that will be used to play the partition.
    * @param {AudioContext} audioContext - The audio context that will be used to play the partition.
    */
   constructor(
-    public json: JSONNote[],
+    public json: JSONMusic[],
     public oscillator: OscillatorType = "sine",
     public audioContext: AudioContext,
   ) {
-    enum NoteEnum {
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I removed this from here, it was redundant with the implementation in Note